src/util/domain-icon.js

```diff
@@ -38,6 +38,9 @@ export default function domainIcon(domain, state) {
     case 'lock':
       return state && state === 'unlocked' ? 'mdi:lock-open' : 'mdi:lock';
 
+    case 'garage_door':
+      return state && state === 'open' ? 'mdi:arrow-up' : 'mdi:arrow-down';
```

Comment:
The problem with that, is I think we need a different icon for open and closed. It may be confusing having the same icon for both states. Maybe window-open and window-closed?